models.js

```diff
@@ -21,13 +21,21 @@ exports.findReviewByID = (id) => {
 exports.postReviewComment = (id, data) => {
   const review_id = parseInt(id.review_id);
   const { username, body } = data;
-
-  return connection
-    .query(
-      `INSERT INTO comments (author, body, review_id) VALUES ($1, $2, $3) RETURNING *`,
-      [username, body, review_id]
-    )
-    .then((res) => {
-      return res.rows;
-    });
+  if (
+    !data.hasOwnProperty("username") ||
+    !data.hasOwnProperty("body") ||
+    typeof data.body != "string" ||
```

Comment:
Everything coming back from psql will be a string, even if you send a json body of {"body": 2}  I get that you're checking for data types, but the table constraints in the seed file will do this for you. If the body column is a text or varchar it is going to kick off at an integer.